app/middleware/cache_headers.py

```diff
@@ -26,9 +30,13 @@ def __call__(self, request):
 
         # Code to be executed for each request/response after
         # the view is called.
-
-        if request.path in ['/checker', '/get-token']:
-            # never cache the /checker and /get-token endpoints.
+        # match /xxx or /stac/api/xxx
+        if (
+            re.match(r'^/((%s/)\w+|\w+)$' % STAC_BASE, request.path) or
+            # DOTO: /get-token could be moved to the base like f.ex /checker
```

Comment:
it could be moved to `/api/get-token` (it must be publicly visible). Imho we could do this change, GDWH has a token now, so not critical for them

Reply:
@boecklic that was my idea to leave it `get-token` as it was to not enter in conflict with GDWH but if you think we are all good then, we should do it :+1:

Reply:
ok